front/src/views/Boxes/components/BoxesActionsAndTable.tsx

```diff
@@ -251,24 +251,24 @@ function BoxesActionsAndTable({
   );
 
   // Unassign tags from boxes
-  // const { unassignTags, isLoading: isUnassignTagsLoading } = useUnassignTags();
-  // const onUnassignTags = useCallback(
-  //   async (tagIds: string[]) => {
-  //     await unassignTags(
-  //       selectedBoxes.map((box) => box.values.labelIdentifier),
-  //       tagIds.map((id) => parseInt(id, 10)),
-  //     );
-  //   },
-  //   [unassignTags, selectedBoxes],
-  // );
+  const { unassignTags, isLoading: isUnassignTagsLoading } = useUnassignTags();
+  const onUnassignTags = useCallback(
+    async (tagIds: string[]) => {
+      await unassignTags(
```

Comment:
Can you add an early-exit here or better into the underlying hook in the case of `tagIds` being empty. In this case we can avoid sending a request to the BE that has no effect anyways.